frame/pablo/src/lib.rs

```diff
@@ -278,6 +289,9 @@ pub mod pallet {
 		WeightsMustBeNonZero,
 		WeightsMustSumToOne,
 		StakingPoolConfigError,
+		NoAvailableLPtokensForSingleAssetWithdraw,
+		NotEnoughLpTokenForSingleAssetWithdraw,
+		NoneValueInLpTokenForSingleAssetWithdraw,
```

Comment:
not sure what this means; perhaps "LpTokenNotFoundForSingleAssetWithdrawal" or something similar?

Reply:
Yes, you right. Added  `LpTokenNotFoundForSingleAssetWithdrawal` instead `NoAvailableLPtokensForSingleAssetWithdraw` and `NoneValueInLpTokenForSingleAssetWithdraw`.